### ISSUE TITLE
Add min/max image dimentions

### ISSUE THREAD

**Issue by jackttcoms** [offboard/php-upload#5]
A possible improvement is add ability to only allow images 200x200 etc.

Also chunked uploads (bit by bit, put together at end) would your class be able to support this? https://github.com/ArviZ/BigUpload

Thanks

**Comment by brunoribeiro94** [comment]
No, yet is not possible, I liked your idea. maybe I'll implement when I've free time.

**Comment by jackttcoms** [comment]
Will you add this in the future? Thanks

**Comment by brunoribeiro94** [comment]
@jackttcoms I know do it, but I working in a project, I no have time now. I have not had donations yet to encourage improvement.

**Comment by jackttcoms** [comment]
Ah ok thanks. 👍  Other than that.... amazing class :)
